apps/studio/prisma/scripts/createSite.ts

```diff
@@ -0,0 +1,236 @@
+    items: [
+      {
+        name: "PA's network one",
+        url: "/item-one/pa-network-one",
+        description: "Click here and brace yourself for mild disappointment.",
+      },
+      {
+        name: "PA's network two",
+        url: "/item-one/pa-network-two",
+        description: "Click here and brace yourself for mild disappointment.",
+      },
+      {
+        name: "PA's network three",
+        url: "/item-one/pa-network-three",
+      },
+      {
+        name: "PA's network four",
+        url: "/item-one/pa-network-four",
+        description:
+          "Click here and brace yourself for mild disappointment. This one has a pretty long one",
+      },
+      {
+        name: "PA's network five",
+        url: "/item-one/pa-network-five",
+        description:
+          "Click here and brace yourself for mild disappointment. This one has a pretty long one",
+      },
+      {
+        name: "PA's network six",
+        url: "/item-one/pa-network-six",
+        description: "Click here and brace yourself for mild disappointment.",
+      },
+    ],
+  },
+]
+const FOOTER_ITEMS = [
+  {
+    title: "About us",
+    url: "/about",
+  },
+  {
+    title: "Our partners",
+    url: "/partners",
+  },
+  {
+    title: "Grants and programmes",
+    url: "/grants-and-programmes",
+  },
+  {
+    title: "Contact us",
+    url: "/contact-us",
+  },
+  {
+    title: "Something else",
+    url: "/something-else",
+  },
+  {
+    title: "Resources",
+    url: "/resources",
+  },
+]
+
+export const createSite = async (siteName: string) => {
+  const siteId = await db.transaction().execute(async (tx) => {
+    const { id: siteId } = await tx
+      .insertInto("Site")
+      .values({
+        name: siteName,
+        theme: jsonb({
+          colors: {
+            brand: {
+              canvas: {
+                alt: "#bfcfd7",
+                default: "#e6ecef",
+                inverse: "#00405f",
+                backdrop: "#80a0af",
+              },
+              interaction: {
+                hover: "#002e44",
+                default: "#00405f",
+                pressed: "#00283b",
+              },
+            },
+          },
+        }),
+        config: jsonb({
+          theme: "isomer-next",
+          siteName: "MTI",
+          logoUrl: "https://www.isomer.gov.sg/images/isomer-logo.svg",
+          search: undefined,
+          isGovernment: true,
+        }),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("name")
+          .doUpdateSet((eb) => ({ name: eb.ref("excluded.name") })),
+      )
+      .returning("id")
+      .executeTakeFirstOrThrow()
+
+    await tx
+      .insertInto("Footer")
+      .values({
+        siteId,
+        content: jsonb({
+          contactUsLink: "/contact-us",
+          feedbackFormLink: "https://www.form.gov.sg",
+          privacyStatementLink: "/privacy",
+          termsOfUseLink: "/terms-of-use",
+          siteNavItems: FOOTER_ITEMS,
+        }),
```

Comment:
nit: maybe can put this together with the `FOOTER_ITEMS`? Found that there were a couple of instances I had to override this

Reply:
done